driver-core/src/test/unit/com/mongodb/internal/operation/AsyncQueryBatchCursorSpecification.groovy

```diff
@@ -27,6 +27,7 @@ import com.mongodb.connection.ServerConnectionState
 import com.mongodb.connection.ServerDescription
 import com.mongodb.connection.ServerType
 import com.mongodb.connection.ServerVersion
+import com.mongodb.internal.async.SingleResultCallback
```

Comment:
The changes in `AsyncQueryBatchCursorSpecification` were done to make unit tests related to calling `close` concurrently with other operations more in line with the corresponding unit tests in `QueryBatchCursorSpecification`.